Story/OnlineUIComponents/OnlinePlayerDisplay.cs

```diff
@@ -105,6 +108,25 @@ public override void Update()
                 }
 
                 this.counter++;
+
+                if (this.label.text != customization.nickname) // we've updated a username
+                {
+                    resetUsernameCounter--;
+                    Color currentColor = this.label.color;
+                    float blendFactor = 0.9f;
+
+                    Color closerToWhite = Color.Lerp(currentColor, Color.white, blendFactor);
+                    this.label.color = closerToWhite;
```

Comment:
Look into recoloring closer to white / just add white. This lerp doesn't do anything.